### PR TITLE
Update make_var_mx6ul_dart_debian.sh

### PR DESCRIPTION
**Pull request by stdcerr**
typo fix and removed obsolete space in path to invoke third-stage script

### CODE REVIEW

File: make_var_mx6ul_dart_debian.sh
Comment (leoniloris) [inline make_var_mx6ul_dart_debian.sh:454]: This will not run third_stage, but instead, it will try to open /bin/sh using ${ROOTFS_BASE}/third-stage as root filesystem, I believe, that's not the intention here. I might be wrong though

**Review by leoniloris — CHANGES_REQUESTED** [review]

File: make_var_mx6ul_dart_debian.sh
Comment (stdcerr) [reply make_var_mx6ul_dart_debian.sh:454]: I got a bunch of errors when the script got to third stage about it being unable to find `apt-get` so I launched the `third-stage` script separately and it worked fine. Then went fixed this path in the script above and ran it without any problems... but I see what you're saying, I might be wrong with that.
Just tried again:
With `LANG=C chroot ${ROOTFS_BASE} /third-stage` I get:
```
/third-stage: line 13: apt-get: command not found

###########################
## Fix missing packages ###
###########################

/third-stage: line 13: apt-get: command not found
```
it goes away and runs fine with `LANG=C chroot ${ROOTFS_BASE}/third-stage` though

Comment (leoniloris) [reply make_var_mx6ul_dart_debian.sh:454]: 👍 I'll just wait for one of the maintainer's feedback

Comment (hj2006) [reply make_var_mx6ul_dart_debian.sh:454]: @stdcerr we tested your suggestion, but it doesn't produce any of the error mentioned above. Could you please share your complete build details and build machine details? It seems like you may be using a different environment. 
We do all tests on the builds and verify the build before publishing the release/changes. 
You can always log your question/release on our customer portal support. 
Variscite customer portal is our official place for technical questions and you will get faster answers there.
https://varisciteportal.axosoft.com/

Comment (hj2006) [reply make_var_mx6ul_dart_debian.sh:454]: @stdcerr 
I am unable to re-produce the issue. 
It seems your build environment could be the factor. 
I am closing this issue for now. If you have further questions, please create a ticket at https://varisciteportal.axosoft.com/

**Pull request closed without merging**
